### PR TITLE
Re-enable OpenSUSE install test

### PR DESCRIPTION
**Pull request by rmartin16**
## Changes
- Closes beeware/briefcase#1802

This reverts commit 66a27a9779b2f57e8adcec17dbd0eac91d006d24.

## PR Checklist:

- [x] All new features have been tested
- [x] All new features have been documented
- [X] I have read the **CONTRIBUTING.md** file
- [X] I will abide by the code of conduct

### CODE REVIEW

**Comment by rmartin16** [comment]
The RPM is no longer erroring at install...

**Review by freakboy3742 — APPROVED** [review]
Looks good; will wait until #150 has been merged so that main doesn't have any failures.

**Pull request merged**
